style/outsidein.css

```diff
@@ -90,15 +90,26 @@ body {
 }
 
 .outside-in-iconColumn1:before {
-    content: "\e606";
+    content: "\e601";
 }
 
 .outside-in-container .outside-in-iconColumn2:before {
-    content: "\e604";
+    content: "\e603";
 }
 
 .outside-in-container .outside-in-iconColumn3:before {
-    content: "\e600";
+    content: "\e607";
+}
+
+.outside-in-container .outside-in-infoRow1 h3:before, 
+.outside-in-container .outside-in-infoRow2 h3:before,
+.outside-in-container .outside-in-aboutPartnersRow h3:before {
+    background: #C0D660;
```

Comment:
Actually, Jon has a pull request that fixes this – and other contrast theme issues. I'll see about merging that too, while I'm at it.